app/src/main/java/com/example/coinage/fragments/AddFragment.java

```diff
@@ -31,9 +31,9 @@
 public class AddFragment extends Fragment {
     public static final String TAG = "AddFragment";
 
-    final Calendar myCalendar= Calendar.getInstance();
-    String myFormat="MM/dd/yy";
-    SimpleDateFormat dateFormat = new SimpleDateFormat(myFormat, Locale.US);
+    public static final Calendar myCalendar = Calendar.getInstance();
```

Comment:
Sorry, only the constants need to be public static. Others  need not be static -- just public/private final.